charts/nri-metadata-injection/values.yaml

```diff
@@ -16,8 +16,8 @@ image:
     # - name: regsecret
 
 jobImage:
-  repository: jettech/kube-webhook-certgen
-  tag: v1.5.0
+  repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
```

Comment:
this was used also in the newrelic-infra operator, has it been fixed there already?

Reply:
https://github.com/newrelic/helm-charts/blob/65447698c9798a0b1ead680788de3a2804885f92/charts/newrelic-infra-operator/values.yaml#L26

Reply:
Afaik the PR @invidian made to support signing other objects is still pending, but this one should work for 1.22.

Reply:
Yeah, this one works for 1.22.